libs/core-wrappers/jest.config.ts

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable */
+export default {
+  displayName: 'core-components',
+  preset: '../../jest.preset.js',
+  transform: {
+    '^(?!.*\\.(js|jsx|ts|tsx|css|json)$)': '@nrwl/react/plugins/jest',
+    '^.+\\.[tj]sx?$': 'babel-jest',
+  },
+  moduleFileExtensions: ['ts', 'tsx', 'js', 'jsx'],
+  coverageDirectory: '../../coverage/apps/tup-ui',
```

Comment:
I am unsure of my suggestion here:
```suggestion
  coverageDirectory: '../../coverage/libs/core-wrappers',
```